samples/README.md

```diff
@@ -0,0 +1,88 @@
+# TerraCost samples
```

Comment:
IDK if this is the clearest way to do this, while reading it I have the feeling this are too separated and hard to understand.

I would try to write 1 example that uses everything and play with flags to opt in/out and pass file paths instead of the user having to modify things manually to run the examples.

IDK I just find it complex

Reply:
If not too complex to read, yes we could go with flags opt for paths and the 3 usecases in the same example